test/functional-tests/app.spec.js

```diff
@@ -7,12 +7,12 @@ describe('Portal application', () => {
       })
     })
   })
-  describe('Nav bar', () => {
-    it("should render the user's PTI balance", () => {
-      browser.url('http://localhost:8080')
-      browser.waitUntil(() =>
-        browser.getText('[data-test-id="nav-pti-balance"]').includes('PTI')
-      )
-    })
-  })
+  // describe('Nav bar', () => {
+  //   it("should render the user's PTI balance", () => {
+  //     browser.url('http://localhost:8080')
+  //     browser.waitUntil(() =>
+  //       browser.getText('[data-test-id="nav-pti-balance"]').includes('PTI')
+  //     )
+  //   })
+  // })
```

Comment:
Do we need to fix these tests?

Reply:
now when you arrive to the app there is no more the `PTI balance`, so this test don't make sense

Reply:
Can we delete them then?